lib/broccoli/angular-broccoli-sass.js

```diff
@@ -24,26 +25,32 @@ class SASSPlugin extends Plugin {
   }
 
   build() {
-    this.listEntries().forEach(e => {
-      let fileName = path.resolve(e.basePath, e.relativePath);
+    this.listFiles().forEach(fileName => {
       this.compile(fileName, this.inputPaths[0], this.outputPath);
     });
   }
 
   compile(fileName, inputPath, outputPath) {
-    let sassOptions = {
-      file: path.normalize(fileName),
-      includePaths: this.inputPaths
-    };
+    const outSourceName = fileName.replace(inputPath, outputPath);
+    const outFileName = outSourceName.replace(/\.s[ac]ss$/, '.css');
 
-    let result = sass.renderSync(sassOptions);
-    let filePath = fileName.replace(inputPath, outputPath).replace(/\.s[ac]ss$/, '.css');
+    const sassOptions = {
+      file: fileName,
+      outFile: outFileName,
+      includePaths: [inputPath].concat(this.options.additionalPaths || [])
+    };
 
```

Comment:
You could do here:
```js
let sassOptions = {
  file: fileName,
  outFile: outFileName,
  includePaths: [inputPath].concat(this.options.additionalPaths || [])
};

sassOptions = Object.assign(sassOptions, this.options);
```

since there are many other options that can be set ([examples](https://github.com/sass/node-sass#examples)).

What do you think?